cirq/protocols/commutes_protocol.py

```diff
@@ -185,31 +184,4 @@ def _strat_commutes_from_matrix(
         return NotImplemented
     if v1.shape != v2.shape:
         return None
-    return linalg.matrix_commutes(v1, v2, atol=atol)
-
-
-def _strat_commutes_from_operation(
-        v1: Any,
-        v2: Any,
-        *,
-        atol: float,
-) -> Union[bool, NotImplementedType, None]:
-    if not isinstance(v1, ops.Operation) or not isinstance(v2, ops.Operation):
-        return NotImplemented
-
-    if set(v1.qubits).isdisjoint(v2.qubits):
-        return True
-
-    from cirq import circuits
-    circuit12 = circuits.Circuit(v1, v2)
-    circuit21 = circuits.Circuit(v2, v1)
-
-    # Don't create gigantic matrices.
-    if np.product(qid_shape_protocol.qid_shape(circuit12)) > 2**10:
-        return NotImplemented  # coverage: ignore
-
-    m12 = unitary_protocol.unitary(circuit12, default=None)
-    m21 = unitary_protocol.unitary(circuit21, default=None)
-    if m12 is None:
-        return NotImplemented
-    return np.allclose(m12, m21, atol=atol)
+    return linalg.matrix_commutes(v1, v2, atol=atol)
```

Comment:
Add newline at the end of this file.

Reply:
This comment is a bit confusing since this code (lines 213:215 from the comment) was moved to ops/raw_types.py:Operation:_commutes_